main.go

```diff
@@ -50,6 +50,14 @@ func main() {
 		return
 	}
 
+        if opts.ExtraJwtIssuers != nil {
```

Comment:
If you really want to implement this in the interim until deprecation, this isn't the appropriate spot.

This belongs in the validation package and likely needs to be flag & alpha options aware? (Hence my recommendation to punt until we completely deprecate this structure).

Reply:
Moved it to validation options.